recommendations/tech_learning.md

```diff
@@ -0,0 +1,77 @@
+# Tech Learning
+We often hear that learning to google (or perhaps [DuckDuckGo](www.duckduckgo.com)) and use stack overflow is are the most important core competencies of any programmer, but growth and learning happen best outside the vacuum of the internet. This page is a list of the engineering team's favorite resources -tutorials, essays, starting points and general thoughts- on a variety of programming topics. 
+
+This is a living document; if you find something out of date or have an addition please open a [pull request](#).
+
+## Good starting points & tools
+
+| Name | Description |
+| -- | -- |
+| [b0rk's zines](https://jvns.ca/zines) | [Julia Evans](https://www.twitter.com/b0rk) makes great comics about *nix programming- [here's a team favorite on learning](https://jvns.ca/wizard-zine.pdf) |
+| [Learn X in Y minutes](https://learnxinyminutes.com/) | Scenic tours of pretty much every language |
+| [Symbol Hound](https://www.symbolhound.com/) | Symbol-aware searching ( e.g. `[ ? { > &` ) |
+
```

Comment:
Julia Evans' work is amazing! Glitch also has a great zine: https://glitch.com/you-got-this